assets/training/model_evaluation/components/model_prediction/spec.yaml

```diff
@@ -90,26 +86,24 @@ outputs:
 
 is_deterministic: True
 code: ../../src
-environment: azureml://registries/azureml/environments/model-evaluation/versions/12
+environment: azureml://registries/azureml/environments/model-evaluation/versions/13
 
 command: >-
   python download_dependencies.py
-  --mlflow-model '${{inputs.mlflow_model}}'
-  $[[--model-uri ${{inputs.model_uri}}]] ;
+  --mlflow-model '${{inputs.mlflow_model}}' ;
   python model_prediction.py
   --data '${{inputs.test_data}}'
-  --task ${{inputs.task}}
+  --task '${{inputs.task}}'
   --predictions '${{outputs.predictions}}'
   --prediction-probabilities '${{outputs.prediction_probabilities}}'
   --ground-truth '${{outputs.ground_truth}}'
   --performance-metadata '${{outputs.performance_metadata}}'
   --mlflow-model '${{inputs.mlflow_model}}'
-  --device ${{inputs.device}}
+  --device '${{inputs.device}}'
   $[[--input-column-names '${{inputs.input_column_names}}']]
-  $[[--model-uri ${{inputs.model_uri}}]]
-  $[[--label-column-name ${{inputs.label_column_name}}]]
+  $[[--label-column-name '${{inputs.label_column_name}}']]
```

Comment:
Keep single quotes

Reply:
I've added the single quotes, not removed them.